forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -6,10 +6,21 @@ import com.joinforage.forage.android.core.element.StatefulElementListener
 import com.joinforage.forage.android.core.element.state.ElementState
 
 /**
- * Configuration details required by ForageElement to operate correctly
+ * The configuration details that Forage needs to create a functional [ForageElement].
  *
- * @property merchantId Your merchant FNS number
- * @property sessionToken A temporary token used for authenticate
+ * Pass a [ForageConfig] instance in a call to
+ * [setForageConfig][com.joinforage.forage.android.ui.AbstractForageElement.setForageConfig] to
+ * configure an Element.
+ *
+ * @property merchantId Either a unique seven digit numeric string that
```

Comment:
Can we emphasize the `mid/<merchant-id>` format (from the Forage dashboard) and mention that the ID can be retrieved from the dashboard?

The accepted formats are:

- `mid/<merchant-id>` (has to be prefixed with `mid/`)
- `<fns-number>`